session/session.go

```diff
@@ -3772,6 +3778,20 @@ func logGeneralQuery(execStmt *executor.ExecStmt, s *session, isPrepared bool) {
 			zap.Bool("isPessimistic", vars.TxnCtx.IsPessimistic),
 			zap.String("sessionTxnMode", vars.GetReadableTxnMode()),
 			zap.String("sql", query))
+		zap.Bool("in-txn", vars.InTxn())
+		zap.Bool("explicit", vars.TxnCtx.IsExplicit)
+		zap.Bool("auto-commit", vars.IsAutocommit())
```

Comment:
```suggestion
			zap.String("sql", query)
			zap.Bool("in-txn", vars.InTxn())
			zap.Bool("explicit", vars.TxnCtx.IsExplicit)
			zap.Bool("auto-commit", vars.IsAutocommit()))
```
Just noticed it's out of the brace of the log function